### DISCUSSION TITLE
All beatmaps deleted after update

### DISCUSSION THREAD

**Discussion by AldisBarbelis** [ppy/osu#30730] [Q&A] [answered]
After I updated osu, all of my beatmaps seem to be gone. Not all data was reset however - I was still logged into my account. I have attached all of the most recent log files. Thanks in advance!
[173170396.zip](https://github.com/user-attachments/files/17792306/173170396.zip)

**Comment by ChromaCat248** [comment]
seconded, also experienced using sh.ppy.osu flatpak on arch linux, along with some notification that said that the database was too new for this version

here are my logs: [compressed-logs.zip](https://github.com/user-attachments/files/17792992/compressed-logs.zip)

**Comment by ChromaCat248** [comment]
are you on the flatpak? if so then yep, the latest flatpak included the wrong version for some reason, `flatpak info sh.ppy.osu` says it's 2024.1115.2 but the debug section says 2024.1009.1

for now until the flatpak is fixed you can use [this workaround](https://github.com/flatpak/flatpak/issues/3097#issuecomment-530583469) to install the previous version (run `sudo flatpak update sh.ppy.osu --commit=72a1e093597a8d84ea098d5051c923eb5e8c4eda43a0bd04f1415267b8f8c3a5`, sudo is needed because flatpak needs root permissions to upgrade to a specific commit)

**Reply by peppy** [reply, marked as answer]
Sorry about that. Should be fixed now (please update your flatpacks).
